### ISSUE TITLE
Add paging for singleColumnMenu or gridMenu?

### ISSUE THREAD

**Issue by ahutchinson1** [cronvel/terminal-kit#91]
The paging ability for singleLineMenu makes it particularly useful. So useful that I use it exclusively, even when singleColumnMenu or gridMenu would make much better use of space. Is it possible to add paging to either of the other menus?

**Comment by cronvel** [comment]
@ahutchinson1 Yes that's an useful feature, it's on my TODO already, but lack the time to do it now.
